health/service_test.go

```diff
@@ -0,0 +1,59 @@
+package health
+
+import (
+	"context"
+	"testing"
+
+	"github.com/projectcalico/app-policy/proto"
+)
+
+type reporter struct {
+	Ready bool
+}
+
+func (r *reporter) Readiness() bool {
+	return r.Ready
+}
+
+func TestHealthService(t *testing.T) {
+	// Test happy path case where ReadinessReporter reports true.
+	reporter := &reporter{
+		Ready: true,
+	}
+	s := NewHealthCheckService(reporter)
+
+	req := &proto.HealthCheckRequest{}
+	resp, err := s.CheckReadiness(context.Background(), req)
+	if err != nil {
+		t.Errorf("expected no error checking readiness, got: %s", err)
+	}
+	if !resp.Healthy {
+		t.Error("expected readiness response to be true")
+	}
```

Comment:
I think you'll find these test assertions to be easier to write and easier to read if you use `gomega` (look at the other UTs in this repo for examples).  For example, these two cases would read something like
```
Expect(err).ToNot(HaveOccurred())
Expect(resp.Healthy).To(BeTrue())
```

Reply:
Ahh yeah, Gomega makes that way more succinct.